tests/Behat/UserContext/UserOperationsContext.php

```
@@ -128,63 +133,42 @@ public function setLanguage(string $lang): void
+    public function theErrorMessageShouldContain(string $partialMessage): void
     {
-        Assert::assertEquals($statusCode, $this->response->getStatusCode());
+        $content = $this->getPageContent();
+        $decoded = json_decode($content, true);
+
+        $errorMessage = '';
+        if ($decoded && isset($decoded['detail'])) {
+            $errorMessage = $decoded['detail'];
+        } else {
+            $errorMessage = $content;
+        }
```

PHPMD (2.15.0)

The method theErrorMessageShouldContain uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)